src/content/glossary/index.md

```diff
@@ -709,6 +858,10 @@ A scaling solution that uses a separate chain with different, often faster, [con
   Sidechains
 </DocLink>
 
+### signing {#signing}
+
+Demonstrating that a transaction originated from a specific person by encoding it using a private key.
+
```

Comment:
Hrm, don't love this definition.
"Signing" more broadly is using the private keys along with an arbitrary message to generate a cryptographic signature that can easily be verified to have come from the private keys behind the public address. You're not really encoding anything, since a signature cannot be decoded to get the original message. Given a public address/key and a message, these can be signed with the private keys, producing a signature, which can be easily verified to confirm that the private keys must have been available to produce the valid signature.
Also, don't love the use of "specific person" here... keys/addresses are not people and people are not keys.

Reply:
fair. updated to: "Demonstrating cryptographically that a transaction was approved by the holder of a specific private key."